### PR TITLE
Feature/add time sync logs

### PR DESCRIPTION
**Pull request by MishkaMN**
# PR Details
## Description

Adds logs to support https://github.com/usdot-fhwa-stol/carma-analytics-fotda/pull/43
## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING**](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: ns3_adapter/src/ns3_adapter.cpp [hunk 1]
```diff
@@ -99,7 +100,12 @@ void NS3Adapter::onDisconnectHandler() {
 void NS3Adapter::onTimeReceivedHandler(unsigned long timestamp)
 {
     rosgraph_msgs::Clock time_now;
-    ROS_DEBUG_STREAM("Received timestamp! " << std::to_string(timestamp));
+    auto chrono_time = std::chrono::system_clock::now();
```
Comment (paulbourelly999) [inline ns3_adapter/src/ns3_adapter.cpp:106]: Nit: Might be helpful to document why we are making this log statement

File: ns3_adapter/src/ns3_adapter.cpp [hunk 2]
```diff
@@ -99,7 +100,12 @@ void NS3Adapter::onDisconnectHandler() {
...
+    auto epoch = chrono_time.time_since_epoch();
+    auto milliseconds = std::chrono::duration_cast<std::chrono::milliseconds>(epoch);
+    auto time_to_milli = static_cast<int>(timestamp / 1e6);
+    ROS_WARN_STREAM("Simulation Time: " << std::to_string(time_to_milli) << " where current system time is: "
```
Comment (paulbourelly999) [inline ns3_adapter/src/ns3_adapter.cpp]: Warn is definitely not the appropriate log level for this.

**Review by paulbourelly999 — CHANGES_REQUESTED** [review]
See comments.

File: ns3_adapter/src/ns3_adapter.cpp (hunk 1 above)
Comment (MishkaMN) [reply ns3_adapter/src/ns3_adapter.cpp:106]: Added document and link to improvement github issue

File: ns3_adapter/src/ns3_adapter.cpp (hunk 2 above)
Comment (MishkaMN) [reply ns3_adapter/src/ns3_adapter.cpp]: Moved to DEBUG as I just learned that I am able to change the level from the same place platform changes levels

**Review by paulbourelly999 — APPROVED** [review]

**Pull request merged**